femitj/iReporter
# #162495462 Create a Sign-up Endpoint for User

**What does this PR do?**

Creates a sign-up endpoint for User account

**Description of the task to be completed**

Ensure the following endpoints are working as they should

POST/api/v1/auth/users

**How should this be tested manually**

* After you clone the repo, 
* cd into it and run npm install to install node modules. 
* Make sure you have nodejs and express installed. 
* Afterwards, run npm start to initiate the app.

test endpoint above using postman with localhost:8080 as the host
sign-up with: email e.g something@mail.com,
                      password,
                      firstname, lastname, username;

**What are the relevant pivotal stories**
[#162495462](https://www.pivotaltracker.com/story/show/162495462)
